src/encoded/static/components/report.js

```diff
@@ -246,22 +247,29 @@ var Table = module.exports.Table = React.createClass({
 });
 
 
-var ColumnSelector = React.createClass({
+var ColumnSelector = createReactClass({
     getInitialState: function() {
         return {
             open: false
         };
     },
 
     render: function() {
+        const columnPaths = Object.keys(this.props.columns);
         return (
             <div style={{display: 'inline-block', position: 'relative'}}>
                 <a className={'btn btn-info btn-sm' + (this.state.open ? ' active' : '')} href="#" onClick={this.toggle} title="Choose columns"><i className="icon icon-columns"></i> Columns</a>
                 {this.state.open && <div style={{position: 'absolute', top: '30px', width: '230px', backgroundColor: '#fff', padding: '.5em', border: 'solid 1px #ccc', borderRadius: 3, zIndex: 1}}>
                     <h4>Columns</h4>
-                    {_.mapObject(this.props.columns, (column, path) => <div onClick={this.toggleColumn.bind(this, path)} style={{cursor: 'pointer'}}>
-                        <input type="checkbox" checked={column.visible} /> {column.title}
-                    </div>)}
+                    {columnPaths.map((columnPath) => {
```

Comment:
No; that `.map()` is an ES6 thing. The problem here was the old code was giving an object to React to render multiple components. That doesn’t work in React 15 — multiple components _have_ to be rendered with an array, and `.map()` returns an array.